core/lib/dr_tools.h

```diff
@@ -342,7 +342,8 @@ typedef enum {
      */
     DR_MEMORY_DUMP_LDMP = 0x0001,
     /**
-     * Memory dump in Executable and Linkable Format.
+     * Memory dump in Executable and Linkable Format. This API has the same
+     * restriction as dr_suspend_all_other_threads_ex().
```

Comment:
Please make it clear it has the same restrictions in *where it can be called from*.  (There are multiple types of restrictions in the suspend docs so it could be confusing.)